internal/tutils/idheader_utils.go

```diff
@@ -41,7 +41,8 @@ var completeIdHeader string = `{
 		 	"is_internal": true,
 		 	"locale": "en-US"
 		},
-		"internal": {"org_id": "%s"}
+		"internal": {"org_id": "%s"},
+		"account_number": "000000"
```

Comment:
We might want to just remove account_number usage completely and keep it null for all records, IB seems to have dropped the usage of it.

Reply:
What do you mean they dropped it? I still can see it somewhere in IB codebase. :thinking:

Reply:
Looks like account is still getting stored, however, it is unused for queries. Looks like some transition phase, to be on the safe side I suggest to still use it. It can be deleted in the final removal patch, I guess that is planned at some point.